metroae

```diff
@@ -1403,6 +1468,45 @@ function download_container {
     fi
 }
 
+function download_container_templates {
+    debug ${FUNCNAME[0]}
+    if ! command -v curl &> /dev/null; then
+        write_to_screen_and_script_log "The command 'curl' was not found in PATH"
+        write_to_screen_and_script_log "This is a small utility for downloading files from the Internet"
+        write_to_screen_and_script_log "It is needed for downloading templates from Amazon S3"
+        write_to_screen_and_script_log "Please install this utility with:"
+        write_to_screen_and_script_log ""
+        write_to_screen_and_script_log "sudo yum install curl"
+        write_to_screen_and_script_log ""
+        exit 1
+    fi
+    curl -O $TEMPLATE_TAR_LOCATION
+    curl -O $VSD_SPECIFICATIONS_LOCATION
+    template_tar_file=$(basename $TEMPLATE_TAR_LOCATION)
+    specs_tar_file=$(basename $VSD_SPECIFICATIONS_LOCATION)
+    template_and_specs_tar_file="metroae_templates_specs.tar"
+    tar -rf $template_and_specs_tar_file $template_tar_file $specs_tar_file
```

Comment:
You will have to put a `set +e` here to prevent the script from stopping on an error with the tar.  Make sure you set it back after with `set -e`